variants/aws-ecs-2/Cargo.toml

```diff
@@ -42,6 +43,7 @@ path = "../variants.rs"
 # core
 release = { path = "../../packages/release" }
 kernel-6_1 = { path = "../../packages/kernel-6.1" }
+netdog = {path = "../../packages/netdog" }
```

Comment:
why is netdog added as a build dependency only on this variant?

Reply:
Somehow this escaped my remove all